openmc/universe.py

```diff
@@ -293,9 +295,15 @@ def find(self, point):
                     return [self, cell] + cell.fill.find(p)
         return []
 
+    # default kwargs that are passed to plt.legend in the plot method below.
+    # If you change this, be sure to change it in the docstring for plot below.
```

Comment:
The docstring doesn't really need to mention it because it will show up in the method signature
```suggestion
```

Reply:
Ah, I didn't know that! Thanks.